docs/sources/tempo/setup/_index.md

```diff
@@ -1,23 +1,32 @@
 ---
-title: Set up a Tempo server or cluster
-menuTitle: Set up a Tempo server or cluster
-description: Learn how to set up a Tempo server or cluster and visualize data
+title: Set up Tempo
+menuTitle: Set up Tempo
+description: Learn how to set up a Tempo server or cluster and visualize data.
 aliases:
 - /docs/tempo/setup
 weight: 300
 ---
 
-# Set up a Tempo server or cluster
+# Set up Tempo
 
-Tempo is available as a pre-compiled binary, a Docker image, and as common OS-specific packaging.
+Tempo can deployed in one of three modes:
 
-This section describes how to set up Tempo on a single Linux node or as a cluster using Kubernetes and Tanka. Tempo can also be set up as a distributed set of services.
+- monolithic
+- scalable monolithic
+- microservices
```

Comment:
Thank you, Mario. I wondered if these terms were correct or outdated because I've seen distributed and single binary used more often than monolithic. If these are the correct updated terms to use, then I'll go through the entire PR and update the terminology so it's consistent. I'll ask on the Tempo channel just to check in and make sure I'm using the correct terms.

Reply:
This issue was resolved in a slack thread. We're using monolithic and microservices to be consistent with MImir and Loki. Also, scalable monolithic is now rolled underneath monolithic.